content/dstu2/condition.md

```diff
@@ -78,7 +78,7 @@ Create Problem Example:
     access-control-max-age → 0
     cache-control → no-cache
     etag → W/"4736135"
-    location → https://fhir.sandboxcernerpowerchart.com/dstu2/9e2aaa6d-3811-4d84-b5f9-93ccf529eefa/Condition/p4736135
+    location → https://fhir.sandboxcerner.com/dstu2/9e2aaa6d-3811-4d84-b5f9-93ccf529eefa/Condition/p4736135
```

Comment:
Updated here and in allergy intolerance with https://github.com/cerner/fhir.cerner.com/pull/91/commits/51f4cb4fa2d9955dd88d12b76363e65587255b56